### PR TITLE
[NCL-7279] Stop logging cancelled builds as ERROR

### PR DESCRIPTION
**Pull request by nikitabugrovsky**
Do not report cancelled builds as ERROR in userLog.

### CODE REVIEW

File: remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java [hunk 1]
```diff
@@ -380,7 +379,42 @@ public BuildRecord storeResult(BuildTaskRef buildTask, Optional<BuildResult> bui
...
-        userLog.error("Build status: {}.", getBuildStatus(buildResult));
+        if (getBuildStatus(buildResult) != null) {
+            String message;
+            switch (getBuildStatus(buildResult)) {
+                case SUCCESS:
+                case NO_REBUILD_REQUIRED:
+                    break;
+                case FAILED:
+                    buildRecordBuilder.status(FAILED);
+                    message = "Build failed.";
```
Comment (michalovjan) [inline remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java:390]: FAILED for emphasis.

**Review by michalovjan — CHANGES_REQUESTED** [review]

**Comment by thescouser89** [comment]
retest this please

File: remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java [hunk 2]
```diff
@@ -380,7 +379,42 @@ public BuildRecord storeResult(BuildTaskRef buildTask, Optional<BuildResult> bui
         errorLog.append(stackTraceWriter.getBuffer());
         buildRecordBuilder.buildLog(errorLog.toString());
 
-        userLog.error("Build status: {}.", getBuildStatus(buildResult));
+        if (getBuildStatus(buildResult) != null) {
```
[the remark this answers is not in the document]
Comment (nikitabugrovsky) [reply remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java:382]: done!

File: remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java (hunk 1 above)
Comment (nikitabugrovsky) [reply remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java]: done!

File: remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java [hunk 3]
```diff
@@ -380,7 +379,42 @@ public BuildRecord storeResult(BuildTaskRef buildTask, Optional<BuildResult> bui
...
+                case NO_REBUILD_REQUIRED:
+                    break;
+                case FAILED:
+                    buildRecordBuilder.status(FAILED);
+                    message = "Build failed.";
+                    userLog.warn(message);
+                    break;
+                case CANCELLED:
+                    buildRecordBuilder.status(CANCELLED);
+                    message = "Build cancelled.";
```
[the remark this answers is not in the document]
Comment (nikitabugrovsky) [reply remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/builder/datastore/DatastoreAdapter.java]: done!

**Comment by nikitabugrovsky** [comment]
retest this please

**Review by michalovjan — APPROVED** [review]

**Pull request merged**